static/js/pm_list.js

```diff
@@ -62,8 +64,16 @@ function remove_expanded_private_messages() {
     resize.resize_stream_filters_container();
 }
 
+function open_private_more_messages() {
+    $(".expanded_private_messages").expectOne().removeClass("zoom-out").addClass("zoom-in");
+    $(".expanded_private_messages li.expanded_private_message").each(function () {
```

Comment:
We also have this CSS:

```
434 .zoom-out .zoom-out-hide {
435     display: none;
436 } 
```

Here's a bit of the template for context:

```
  3     <li class='{{#if is_zero}}zero-subject-unreads{{/if}} {{#if zoom_out_hide}}zoom-out-hide{{/    if}} expanded_private_message' data-user-ids-string='{{user_ids_string}}'>
```

For rows that should be invisible at first (before you click "More") we add the `zoom-out-hide` classes to them.

And when we expand the PM list, we do this:

```
170         $("#global_filters li[data-name='private']").addClass('active-filter zoom-out');
```